addons/mail/static/src/core/common/thread_service.js

```diff
@@ -851,45 +852,60 @@ export class ThreadService {
     }
 
     /**
-     * @param {import("models").Persona} persona
-     * @param {import("models").Thread} [thread]
+     * @param {{ persona: import("@mail/core/persona_model").Persona, thread: import("@mail/core/thread_model").Thread }}
```

Comment:
Keep `import("models").Persona/Thread`. They take patches into account, whereas absolute path to original code do not have patches.

And also using `import("models")` is dryer, and agnostic to arbitrary split in folders that is prone to mistakes (e.g. you forgot `common/` subfolder)